### PR TITLE
netdev: add stubs for get/set not implemented

### PR DESCRIPTION
**Pull request by haukepetersen**
### Contribution description
Selected `netdev` driver implementations do not implement the `get()` and/or `set()` functions of the `netdev` interface. This PR provides two convenience functions for these cases, so there is no need to write the 'return always -ENOTSUP' functions over and over...

For ref: the same approach is also used by SAUL.

### Testing procedure
IMO a successful buildtest should be sufficient, right?!

### Issues/PRs references
none

### CODE REVIEW

File: drivers/include/net/netdev.h [hunk 1]
```diff
@@ -413,6 +414,47 @@ typedef struct netdev_driver {
...
+ *
+ * @param[in] dev           ignored
+ * @param[in] opt           ignored
+ * @param[in] value         ignored
+ * @param[in] max_len       ignored
+ *
+ * @return  always returns `-ENOTSUP`
+ */
+static inline int netdev_get_notsup(netdev_t *dev, netopt_t opt,
+                                    void *value, size_t max_len)
```
Comment (bergzand) [inline drivers/include/net/netdev.h]: Is there any advantage of declaring this inline? It is always used as function pointer right?

Comment (miri64) [reply drivers/include/net/netdev.h]: Not having to create a `netdev` module (and creating dependency relations for the requiring devices) where you would put the implementation ;-). Given that the number of devices that need it is not that large, it isn't worth the effort, but still reduces code duplication.

Comment (haukepetersen) [reply drivers/include/net/netdev.h]: There is no code compiled for the 'netdev' module (as its only a header), so my take was to simply declare it inline, taking the cost of some duplication for the rare case where there is more than one device type using these stubs compiled in.

**Comment by haukepetersen** [comment]
all green -> go

**Pull request merged**